src/Components/Locations.js

```diff
@@ -0,0 +1,74 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+const Locations = () => {
+  const [locations, setLocations] = useState([]);
+  const [showLocations, setShowLocations] = useState(false);
+  useEffect(() => {
+    fetchLocations();
+  }, []);
+  const fetchLocations = async () => {
```

Comment:
Nitpick: Give your functions a little room to breathe. An empty line on either side of them for better readability.